index.js

```diff
@@ -7,12 +7,42 @@ function ask(questionText) {
   });
 }
 
-start();
+//The computer then guesses within the predetermined range
+let computerGuess = (max, min) => {
+  let guess = (min + max) / 2;
+  return Math.floor(guess);
+};
 
+start();
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  let playing = true;
+  let upperBound = 100
+  let lowerBound = 1
+  let currentGuess = computerGuess(upperBound, lowerBound);
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  //The user picks a secret number
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+  console.log("You entered: " + secretNumber);
+  while (playing) {
```

Comment:
You should replace this while conditional statement with something that tells us the purpose of the loop. For example:

```js
While(the user hasn't responded with "yes") {your code here}
```
In addition, playing never changes in your while loop. Which leaves a risk of an infinite loop.